staging/src/BUILD

```diff
@@ -189,6 +189,7 @@ filegroup(
         "//staging/src/k8s.io/client-go/util/retry:all-srcs",
         "//staging/src/k8s.io/client-go/util/testing:all-srcs",
         "//staging/src/k8s.io/client-go/util/workqueue:all-srcs",
+        "//staging/src/k8s.io/cloud-provider:all-srcs",
```

Comment:
Can't comment on the `100755 → 100644` mode changes, but do they belong in this PR?  Maybe send a separate PR?

Reply:
I agree they do not belong in this PR. Will make the change.